src/main/resources/assets/seedmapper/lang/hi_in.json

```diff
@@ -0,0 +1,82 @@
+{
+  "commands.exceptions.dimensionNotSupported": "आयाम \"%s\" समर्थित नही है।",
+  "commands.exceptions.versionNotFound": "संस्करण \"%s\" नही मिला",
+  "commands.exceptions.unknownConfig": "अज्ञात कॉन्फिग \"%s\".",
+  "commands.exceptions.nullPointerException": "\"%s\" null है",
+  "commands.exceptions.invalidDimension": "अमान्य आयाम।",
+  "commands.exceptions.biomeNotFound": "बायोम \"%s\" नही मिली।",
+  "commands.exceptions.structureNotFound": "सरंचना \"%s\" नही मिली",
+  "commands.exceptions.lootNotFound": "आइटम \"%s\" नही मिला",
+  "commands.exceptions.lootItemNotFound": "लूट आइटम \"%s\" नही मिला।",
+  "commands.exceptions.blockNotFound": "ब्लॉक \"%s\" नही मिला।",
+  "commands.exceptions.expectedWithOrWithout": "Expected \"with\" or \"without\".",
```

Comment:
If you could translate this one as well that would be amazing :).

Reply:
Done :D